### PR TITLE
[categories] Improve french

### PR DESCRIPTION
**Pull request by Jean-BaptisteC**

### CODE REVIEW

File: data/categories_cuisines.txt [hunk 1]
```diff
@@ -140,6 +140,7 @@ sk:Hamburger
 
 cuisine-bubble_tea
 en:Bubble Tea|boba tea
+fr: Thé aux perles
```
Comment (biodranik) [inline data/categories_cuisines.txt:143]:
```suggestion
fr:Thé aux perles
```
@vng will the search work properly with such translation/keyword?

File: data/categories_cuisines.txt [hunk 2]
```diff
@@ -863,7 +864,7 @@ cs:Steaková restaurace
 da:Steakhouse
 nl:Steakhouse
 fi:Pihviravintola
-fr:Grillades
```
Comment (meenbeese) [inline data/categories_cuisines.txt:866]: Grillades, pourquoi ça ne marche pas? Ceci me semble plus naturel en fait.

File: data/categories_cuisines.txt (hunk 1 above)
Comment (vng) [reply data/categories_cuisines.txt:143]: Hm, good case to check when 'the' is a stopword in En, but meaningful in Fr.

**Comment by Jean-BaptisteC** [comment]
Ready to merge

**Comment by vng** [comment]
@Jean-BaptisteC Did you try "Thé aux perles" ? Is it fine? Or only "The"

**Comment by Jean-BaptisteC** [comment]
fr -> Thé = en -> Tea

**Comment by Jean-BaptisteC** [comment]
Yes, search works correctly

**Review by vng — APPROVED** [review]

**Pull request merged**